glossary.md

```diff
@@ -1,5 +1,83 @@
 # Building C++
 
+[backtrace](https://github.com/mapbox/cpp/blob/master/glossary.md#backtrace)
```

Comment:
@GretaCB You might be able to use just the has `#backtrace` instead of the whole URI to make this work across branches!